third_party/onnx-mlir.cmake

```diff
@@ -0,0 +1,18 @@
+include(ExternalProject)
+
+set(ONNX_MLIR_VERSION "67ea9b55f3a55cf4e7f351c56ebf54fda7bbb365")
+
+ExternalProject_Add(
+    onnx-mlir
+    PREFIX ${TTMLIR_TOOLCHAIN_DIR}
+    CMAKE_GENERATOR Ninja
+    CMAKE_ARGS
+    -DCMAKE_CXX_COMPILER=/usr/bin/c++
+    -DPython3_ROOT_DIR=$pythonLocation
```

Comment:
I'm not sure this line is legal cmake syntax.  I think `$pythonLocation` refers to some environment variable the suggest setting in their build docs, but they were running a bash command, I think this line gets evaluated by cmake before it constructs the underlying shell string.